### DISCUSSION TITLE
Feature Request: `word_separators` configuration

### DISCUSSION THREAD

**Discussion by btrazzini** [zed-industries/zed#15801] [Feature Requests]
I would like to suggest the addition of the configuration which I frequently use in VS Code.

```json
{"editor.wordSeparators": "`~!@#%^&*()=+[{]}\\|;:'\",.<>/?-"}
```

This configuration allows me to efficiently select and delete text without invalidating the separators. The ability to customize word separators is crucial for my productivity, and I believe it would benefit other Zed Editor users as well.

**Comment by xx-moos** [comment]
Good advice, yes this does work and there is also a configuration option to collapse all code

**Comment by ArcaneCrowA** [comment]
When i used vscode I also added _ it helps when editing python code (or any snakecase)

**Comment by AmbitionsXXXV** [comment]
It looks like this configuration item isn't supported yet. Is that correct? I would really appreciate having this configuration available as well!

**Comment by rikkarth** [comment]
make this happen please

**Comment by aizigao** [comment]
also need this

**Comment by hammaren26** [comment]
+1, need this for $variable selection

**Comment by martindrzka** [comment]
+1 for this feature 🙏
